homeassistant/components/tractive/config_flow.py

```diff
@@ -17,7 +17,9 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-STEP_USER_DATA_SCHEMA = vol.Schema({CONF_EMAIL: str, CONF_PASSWORD: str})
+STEP_USER_DATA_SCHEMA = vol.Schema(
```

Comment:
Maybe rename this constant since we're using it in the reauth confirm step too now.